### ISSUE TITLE
Add manual override to walltime

### ISSUE THREAD

**Issue by andrewkho** [lanpa/tensorboardX#205]
Hi, I'm using tensorboardX for a slightly unusual purpose: I'm using it to visualize data made from *simulations*, not for neural network training. TensorboardX + Tensorboard is very close to what I currently need for this purpose, however one feature I'd like is to manually override the "walltime" so that I can view the time in my simulation, instead of the actual time. 

I'm looking at the code and don't think it'd be too hard, if I made a Pull Request would you be willing to review?

**Comment by lanpa** [comment]
Sure. Please make sure that it doesn't break the original API.

**Comment by andrewkho** [comment]
Hi @lanpa, I have created a PR #207 for this request. For my use case I'm currently using `SummaryWriter.add_scalar` and `SummaryWriter.add_scalars`, for those two cases they are working. I don't anticipate anything wrong with the other cases though.

Let me know what you think, I named the argument `walltime` instead of `timestamp` since it seems more direct, and second, at least for me, timestamp implies `datetime.datetime` (I could be way off base on this). 

thanks!
